File: countries-by-language.py
country = ["Philippines", "Korea", "Japan", "USA"]
language = ["Filipino", "Hangul", "Nihongo", "English"]
pair_array = []
size = len(country)
for count in range(size):
        pair_array.append((country[count], language[count]))
        
print(pair_array)

File: countries-by-language.js
let country = ["Philippines", "Korea", "Japan", "USA"];
let language = ["Filipino", "Hangul", "Nihongo", "English"];
let pairArray = [];

for (let i = 0; i < country.length; i++) {
  pairArray.push([country[i], language[i]]);
}

console.log(pairArray);